### PR TITLE
Add release drafter, browser compat, use tags

### PR DESCRIPTION
**Pull request by timja**
Fixes https://github.com/jenkinsci/dark-theme/issues/18
Fixes https://github.com/jenkinsci/dark-theme/issues/46

### CODE REVIEW

File: .github/workflows/release-drafter.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    # branches to consider in the event; optional, defaults to all
+    branches:
+      - master
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      # Drafts your next Release notes as Pull Requests are merged into "master"
+      - uses: release-drafter/release-drafter@v5
```
Comment (oleg-nenashev) [inline .github/workflows/release-drafter.yml:14]:
```suggestion
      - uses: release-drafter/release-drafter@v5.9.0
```

File: .github/workflows/release-drafter.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        with:
+          # (Optional) specify config name to use, relative to .github/. Default: release-drafter.yml
+          # config-name: my-config.yml
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (oleg-nenashev) [inline .github/workflows/release-drafter.yml:19]:
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```

File: README.md [hunk 3]
```diff
@@ -1,16 +1,37 @@
 # Dark Theme for Jenkins (Incubated project)
 
-[![Gitter](https://badges.gitter.im/jenkinsci/hackfest.svg)](https://gitter.im/jenkinsci/hackfest?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+[![Gitter](https://badges.gitter.im/jenkinsci/ux-sig.svg)](https://gitter.im/jenkinsci/ux-sig?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-This repository contains a new dark theme for Jenkins which is currently **under development**.
-It requires changes in the Jenkins core, and it is not ready for production use.
-Visualization is also not perfect, contributions are welcome!
+This repository contains a dark theme for Jenkins.
```
Comment (oleg-nenashev) [inline README.md:5]: Would it make sense to keep a disclaimer that it is under development?

Comment (timja) [reply README.md:5]: I think it's fine, its in the title of the repo, and it mostly works, at least with core components

**Pull request merged**